benchmarking/platforms/platform_base.py

```diff
@@ -18,8 +18,7 @@
 class PlatformBase(object):
     def __init__(self, tempdir, tgt_dir, platform_util):
         self.tempdir = tempdir
-        self.platform = None
```

Comment:
Since we delete setting self.platform from the base class, maybe we can leave the rest of the code as is. As the value will not be overwritten.

Reply:
Let's keep it there.